src/services/FsData.service.ts

```diff
@@ -119,14 +101,24 @@ class FsData implements ng.IServiceProvider {
       datasourceIds.push(0);
     }
 
-    let requestUrl = `${this._baseUrl}${this._dataUrl}`.replace(/{{datasources}}/g, datasourceIds.join(',')).replace(/{{type}}/g, dataset);
+    var dataUrl = this._baseUrl + this._dataUrl;
+    var config = {
+      params: {
+        "customer_id": this._customerId,
+        "datasources": datasourceIds.join(','),
+        "type": dataset,
+        "from_date": fromDate || null,
+        "to_date": toDate || null,
```

Comment:
`toDate || null` is just to avoid empty params in the final URL. I could have checked it with an if-condition but this looks better